src/pages/PointOfContact/index.tsx

```diff
@@ -244,7 +244,7 @@ function PointOfContact(): JSX.Element {
           alertHeading={
             hasFormRegexErrors
               ? 'There was a problem updating your point of contact information'
-              : 'You must provide all required point of contact information to save and continue'
+              : 'You must provide all required point of contact information to continue to next step'
```

Comment:
Update:
'There was a problem updating your point of contact information'

Reply:
Heyo @natalia-fitzgerald, these are handled in my PR over here:
https://github.com/cfpb/sbl-frontend/pull/855/files